File: app/src/main/java/com/example/jetpackmvvm/sample/data/model/Owner.java
package com.example.jetpackmvvm.sample.data.model;

import androidx.annotation.NonNull;

import com.google.gson.annotations.SerializedName;

public class Owner {

    @SerializedName("login")
    @NonNull
    public final String login;

    @SerializedName("avatar_url")
    public final String avatarUrl;

    @SerializedName("url")
    public final String url;

    public Owner(@NonNull String login, String avatarUrl, String url) {
        this.login = login;
        this.avatarUrl = avatarUrl;
        this.url = url;
    }
}

File: app/src/main/java/com/terry/jetpackmvvm/sample/data/model/Owner.kt
package com.terry.jetpackmvvm.sample.data.model

import com.google.gson.annotations.SerializedName

data class Owner(
    @field:SerializedName("login")
    val login: String,
    @field:SerializedName("avatar_url")
    val avatarUrl: String,
    @field:SerializedName("url")
    val url: String
)